admin/src/templates/message_queue.html

```diff
@@ -0,0 +1,15 @@
+<ul class="nav nav-tabs">
+  <li role="presentation" ui-sref-active="active">
+    <a ui-sref="message-queue.scheduled" translate>Scheduled</a>
+  </li>
+  <li role="presentation" ui-sref-active="active">
+    <a ui-sref="message-queue.due" translate>Due</a>
+  </li>
+  <li role="presentation" ui-sref-active="active">
+    <a ui-sref="message-queue.muted-future" translate>Will not send</a>
+  </li>
+  <li role="presentation" ui-sref-active="active">
+    <a ui-sref="message-queue.muted-past" translate>Did not send</a>
```

Comment:
These tabs don't appear to have translation keys provided. Change the keys to lowercase and dot separated to make the missing translations easier to spot!